src/config/config.yaml

```diff
@@ -0,0 +1,7 @@
+app:
+  port: ':8081'
+db:
+  charactersPath: 'src/resources/characters.csv'
+  mapPath: 'src/resources/map.csv'
+rest:
+  maxPagesByDefault: 1
```

Comment:
This kind of file should end with an empty line.
Also, in most cases, you should use `"` instead of "'"